GOLGI/donations.yml

```diff
@@ -0,0 +1,23 @@
+- ticker: GOLGI
+
+  recipient: 
+    name: Centre for Regenerative Medicine “Stefano Ferrari”
+    address: Via Glauco Gottardi 10 41125 Modena (Italy)
+    tel: +39.059.205811
+    contact: centromedicinarigenerativ@unimore.it
+
+  ada: 180 # <- These must be numbers. Can be decimals though. See below
+  usd: 212.5 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1599026880 
```

Comment:
This timestamp is more than 1 year ago. please use https://www.unixtimestamp.com/ to generate a matching timestamp.

Reply:
Hello, I have update timestamp on the repository.